sites/elcinema.com/elcinema.com.config.js

```diff
@@ -77,7 +77,7 @@ function parseStart(item, initDate) {
 
   time = `${initDate.format('DD/MM/YYYY')} ${time}`
 
-  return dayjs.tz(time, 'DD/MM/YYYY H:mm A', 'Africa/Cairo')
+  return dayjs.tz(time, 'DD/MM/YYYY H:mm A', 'Saudi Arabia/Riyadh')
```

Comment:
As far as I know, there is no such time zone.

https://en.wikipedia.org/wiki/List_of_tz_database_time_zones

Reply:
@ksa-live I think you meant [Asia/Riyadh](https://en.wikipedia.org/wiki/Asia/Riyadh)

Reply:
@mcastellaneta yes that is the time i need